Makefile

```diff
@@ -55,8 +55,16 @@ version-windows: build-windows
 deps:
 	go mod download
 
-# Run acceptance tests
 testacc: get
+	@echo "Running acceptance tests"
 	go test $(TEST) -v $(TESTARGS) -timeout 10m
 
-.PHONY: lint get build version build-linux build-windows build-macos deps version-linux version-windows version-macos testacc
+testacc-cover: get
```

Comment:
What's the harm in always running with coverage enabled? Want to understand if a separate target is needed